template.json

```diff
@@ -40,7 +41,8 @@
       "lint:eslint": "eslint ./src --ext .js,.jsx,.ts,.tsx --cache",
       "fix": "npm-run-all fix:*",
       "fix:eslint": "npm run lint:eslint -- --fix",
-      "fix:prettier": "prettier \"src/**/*.js,jsx,ts,tsx,scss}\" --write"
+      "fix:prettier": "prettier \"src/**/*.js,jsx,ts,tsx,scss}\" --write",
+      "deploy": "cross-env rsync -avPx build/* ${npm_package_name}@<host>:~/public/ --delete-after"
     }
```

Comment:
Can you add:
```"open": "cross-var open-cli https://$npm_package_name.<host>",```

Reply:
We can add it, but I think that we would need a different name. As `open` gives me the impression that I will open the locally built version of the app, instead of the deployed version. Perhaps, `view-deploy` ?